src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1848,7 +1857,7 @@ inline constexpr auto mutate_apply(
  * \example
  * \snippet Test_DataBox.cpp mutate_apply_struct_definition_example
  * This is how to use `mutate_apply` with the above class:
- * \snippet Test_DataBox.cpp mutate_apply_struct_example_with_tags_lists
+ * \snippet Test_DataBox.cpp mutate_apply_struct_example_stateful
  *
  * \see box
```

Comment:
This line is pointless.  I see that it was copied from the other overload, so could you fix that one as well?